src/components/00-materials/story.scss

```diff
@@ -1,9 +1,5 @@
-@import '../../other/pages/utils/accessory-story.scss';
 @import '../../other/pages/utils/canvas-content-padding.scss';
 
 @include canvas-padding;
 @include content-padding;
-
-@include accessory-story-content;
```

Comment:
why was this deleted?

Reply:
Icons and colors don't have a footer and therefore don't need a bottom padding.